app/assets/stylesheets/all/components/_questions.css.scss

```diff
@@ -21,6 +21,14 @@ form.question_form {
 
 div.question_fields {
 
+  textarea:not([id*=_ar]) {
+    direction: ltr;
+  }
+
+  textarea[id*=_ar] {
+    direction: rtl;
+  }  
+
```

Comment:
@ziedAb This is a clever solution! But I worry that it would not work for RTL languages aside from Arabic. Can you think of another way? Perhaps if you even just listed out all the most popular RTL language codes that would cover it. 

For the direction: ltr one you could just take off the `not` part and make that the default, and then the rtl one will supersede it due to specificity, yes?

Reply:
@hooverlunch thank you for your comments ! 
What other right to left languages are you considering to add in the future and what are their codes ? here is the right to left language list from w3.org https://www.w3.org/International/questions/qa-scripts#examples